src/supertux/menu/options_menu.cpp

```diff
@@ -235,6 +236,10 @@ OptionsMenu::OptionsMenu(bool complete) :
     add_toggle(MNID_CHRISTMAS_MODE, _("Christmas Mode"), &g_config->christmas_mode);
   }
 
+  add_toggle(MNID_CONFIRMATION_DIALOG, _("Confirmation Dialog"), &g_config->confirmation_dialog)
+    ->set_help("Confirm aborting level");
```

Comment:
This is a strange place for a linebreak.

Reply:
I saw it on line 198 of the same file and though it was the standard way.

Reply:
Keep it in one line here, for now.